alerts/email_test.go

```diff
@@ -17,13 +17,14 @@ import (
 const (
 	ProjectName     = "Test Project"
 	ProjectOwner    = "testprojowner"
+	ProjectRepo     = "testrepo"
 	ProjectId       = "testproject"
 	VersionRevision = "aaaaaaaaaaaaaaaaaaa"
 	VersionMessage  = "bbbbbbbbbb"
 	BuildName       = "Linux 64"
 	BuildId         = "b1"
 	TaskName        = "mainTests"
-	TaskId          = "t1"
+	TaskId          = "t1!"
```

Comment:
Just added in the most notably special character, this shouldn't matter outside of URLs.